config/htaccess.yaml

```diff
@@ -55,4 +55,66 @@ type: 'redirect'
 code: 301
 outputs:
   - 'before-v1.1': {'/bi-connector': 'https://docs.mongodb.com/bi-connector'}
+---
+
+###
+### Artifacts from the /bi-connector to /bi-connector/<branch> transition
+###
+
+code: 301
+from: '/schema-configuration'
+to: '/v1.1/schema-configuration'
+outputs:
+  - {'/bi-connector': 'https://docs.mongodb.com/bi-connector'}
+type: redirect
+---
+code: 301
+from: '/installation'
+to: '/v1.1/installation'
+outputs:
+  - {'/bi-connector': 'https://docs.mongodb.com/bi-connector'}
+type: redirect
+---
+code: 301
+from: '/components'
+to: '/v1.1/components'
+outputs:
+  - {'/bi-connector': 'https://docs.mongodb.com/bi-connector'}
+type: redirect
+---
+code: 301
+from: '/components/mongobiuser'
+to: '/v1.1/components/mongobiuser'
+outputs:
+  - {'/bi-connector': 'https://docs.mongodb.com/bi-connector'}
+type: redirect
+---
+code: 301
+from: '/components/mongodrdl'
+to: '/v1.1/components/mongodrdl'
+outputs:
+  - {'/bi-connector': 'https://docs.mongodb.com/bi-connector'}
+type: redirect
+---
+code: 301
+from: '/components/mongobischema'
+to: '/v1.1/components/mongobischema'
+outputs:
+  - {'/bi-connector': 'https://docs.mongodb.com/bi-connector'}
+type: redirect
+---
+code: 301
+from: '/faq'
```

Comment:
I feel like it should go to the more current (if pages exist).  I know it's slightly inconsistent since for the components that got removed, but I think if I type in blah/bi-connector  (without a version), I  more often than not would think it would give me the most recent.